e2e/utils/node.go

```diff
@@ -55,3 +57,25 @@ func WaitForNodeHealthyCondition(k8sClient ctrl.Client, node *v1.Node, status v1
 	}, nodeRebootedTimeout, 1*time.Second).Should(Equal(status))
 	return transitionTime
 }
+
+func GetWorkerNodes(k8sClient ctrl.Client) []v1.Node {
+	workers := &v1.NodeList{}
+	selector := labels.NewSelector()
+	reqCpRole, _ := labels.NewRequirement("node-role.kubernetes.io/control-plane", selection.DoesNotExist, []string{})
```

Comment:
NIT: We can use common labels instead of typing it here and afterwards https://github.com/medik8s/common/blob/main/pkg/labels/labels.go#L9

Reply:
done